### ISSUE TITLE
Response message to `-ft set timezone` makes little sense

### ISSUE THREAD

**Issue by j-rodman** [KevinNovak/Friend-Time#70]
The introductory message calls to action:

-> Set your time zone by using the -ft set command!

Since there is no syntax diagram offered etc, I immediately type:

# -ft set America/Los_Angeles

This comes back with:

-> This command can only be used in a server.

At this point I'm trying to figure how to talk to the bot in a server-specific way, and am ratholed re: using the bot.

Possible Improvements to alleviate:

1 - Give more explicit instructions.  "Set your time zone by typing `-ft set`"  or some similar construction.  A command takes arguments.  Something you just ask the user to type is more explicit.

2 - If someone types `-ft set <explicit_TZ>` just work.  I know this is theoretically hard because almost any string is a valid TZ at least from the libc specification, but realistically for your design you could limit this to the well-formed time zones you expect from your map.

**Comment by Davi-the-Mudkip** [comment]
Yeah, that should be improved.

**Comment by KevinNovak** [comment]
I will be adding [Slash Commands](https://blog.discord.com/slash-commands-are-here-8db0a385d9e6) to Friend Time once discord.js (library Friend Time is using) is updated to handle Slash Commands. This should make the syntax more obvious / less likely for the user to mess up.

**Comment by Davi-the-Mudkip** [comment]
Yeah but don't forget that slash commands are still entirely unfinished, even though Discord has "released" them.
